contentcuration/contentcuration/frontend/channelList/views/Channel/CatalogList.vue

```diff
@@ -46,60 +46,55 @@
             <ChannelItem
               :channelId="item.id"
               :detailsRouteName="detailsRouteName"
-              style="flex-grow: 1; width: 100%;"
+              style="flex-grow: 1; width: 100%"
             />
           </VLayout>
         </VFlex>
-        <VFlex xs12 style="padding-bottom: 72px;">
+        <VFlex xs12 style="padding-bottom: 72px">
           <VLayout justify-center>
-            <Pagination
-              :pageNumber="page.page_number"
-              :totalPages="page.total_pages"
-            />
+            <Pagination :pageNumber="page.page_number" :totalPages="page.total_pages" />
           </VLayout>
         </VFlex>
       </VLayout>
-      <BottomToolBar
+      <BottomBar
         v-if="selecting"
-        clipped-left
-        color="white"
-        flat
         data-test="toolbar"
         :height="$vuetify.breakpoint.xsOnly ? '72px' : '56px'"
```

Comment:
This height prop isnt working now, and this is happening on mobile screens, the bottom bar now doesnt have an appropiate height:

Before:
![image](https://github.com/learningequality/studio/assets/51239030/d0c27a6a-4696-4469-8020-2d802610aaba)

After:
![image](https://github.com/learningequality/studio/assets/51239030/a2348fcc-18da-4653-babe-c7a6d9e48556)

One thing we can do is that the `BottomBar` component receives an appearanceOverrides, where we can pass overrides styles to the bar, and we can end with something like:
`<BottomBar :appearanceOverrides=" { height: ... } ">`

Reply:
I have made the desired changes and tested it on the screen above.